gnomad_qc/v4/annotations/Dockerfile_vrs084_v4

```diff
@@ -53,6 +58,17 @@ RUN wget https://github.com/vcftools/vcftools/releases/download/v${VCFTOOLS_VERS
 	&& make install \
 	&& make clean
 
+# Install bgzip
+ENV BGZIP_VERSION="1.17"
+RUN wget https://github.com/samtools/htslib/releases/download/1.17/htslib-1.17.tar.bz2 \
+    && tar -xjf htslib-1.17.tar.bz2 \
+    && rm htslib-1.17.tar.bz2 \
+    && cd htslib-1.17 \
+    && ./configure \
+    && make \
+    && make install \
+    && make clean \
+    && export PATH=htslib-1.17/bin:$PATH
```

Comment:
since we set the `BGZIP_VERSION` environment variable, we should use it in all the lines that reference that version, so that we only have to update that one line of code when there's a new version. So, replace all the `1.17`s in the `RUN` command with `${BGZIP_VERSION}`